README.md

```diff
@@ -62,7 +62,7 @@ To build the Riak Java Client, you must have the correct version of the riak_pb
 
 ```
 git clone https://github.com/basho/riak_pb
-git checkout java-2.1.1.0
+git checkout 2.1.2.0
```

Comment:
After merging1.3 branch into this one, @aleksey-suprun can prove this conversion by running (or extending, if needed) our Full Bucket Read Test